src/content/docs/release-notes/mobile-release-notes/android-ndk-release-notes/android-ndk-101.mdx

```diff
@@ -0,0 +1,14 @@
+---
+subject: Android NDK Agent
+releaseDate: '2023-04-06'
+version: 1.0.1
+---
+
+## Fixed
+
+* Fixed an ANR issue when using our android NDK agent
```

Comment:
Not sure if I agree _fixed_ is the proper term. More like "addressed a memory leak resulting from ANR detection"